Source/Core/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_subdirectory(AudioCommon)
 add_subdirectory(Common)
 add_subdirectory(Core)
 add_subdirectory(DiscIO)
-add_subdirectory(DolphinWX)
+if(NOT DISABLE_WX)
```

Comment:
It's actually clearer when this is a simple list of subfolders with no extra logic.
Then, the logic specific to each target can disable them. The condition here is simple, but that's not always true, so for consistency, it's better to keep everything tight together in the same file.